### PR TITLE
feat: Fetch sunrise bills/categories instead of hard-coded options

### PR DESCRIPTION
**Pull request by elliott-king**
- [x] add missing bills to airtable

This fetches the bill IDs & campaigns from the airtable, and has a new endpoint to show them. _Only backend_, no frontend implementation atm.

### To test:

Make sure you have `AIRTABLE_API_KEY` in your `.env` file (see directions in #31). Run & visit http://localhost:5173/api/v1/bills/airtable-bills. Verify that:
- each is keyed by a senate or assembly bill ID
- `A01263` has two campaigns

### Reference:

airtable table: https://airtable.com/appBS7e3igXNBXl9b/tblCJq2YX259pH5ij/viwMwbZUFjBlA9KLK
airtable bespoke table API reference: https://airtable.com/appBS7e3igXNBXl9b/api/docs
airtable base API reference: https://airtable.com/developers/web/api/introduction

-----------------

### DEPRECATED - kept for reference only

- [x] remove google sheets logic
- [x] remove google sheets js library
- [x] remove google sheets env var

file:///Users/mymac/Developer/oss/google/google-api-nodejs-client/docs/sheets/classes/Resource$Spreadsheets$Values.html#get
https://developers.google.com/sheets/api/guides/concepts
https://developers.google.com/sheets/api/quickstart/nodejs
https://developers.google.com/sheets/api/guides/values#node.js

### CODE REVIEW

**Comment by agreffard** [comment]
Hey @elliott-king , that's actually really cool!
I wonder how "robust" it is to use in production though? It looks easily breakable is the spreadsheet changes a little for example.
I think @afnanhaq is working on fnalizing the categories on Airtable.
That's a great topic for tomorrow's meeting

File: backend/src/airtable-api.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+export const fetchSunriseBills = async () => {
+  let bills = {};
+  let campaigns = {};
+  try {
+    bills = await sunriseBills();
+    campaigns = await sunriseCampaigns();
+  } catch (err) {
+    console.error('fetch airtable bills or campaigns error', err);
+  }
+  return { bills, campaigns };
```
Comment (elliott-king) [inline backend/src/airtable-api.js:146]: Not a fan of this return formatting, but there's no reason to break this into two endpoints.

Comment (agreffard) [reply backend/src/airtable-api.js:146]: Looks good to me. Especially if you add a slight change to execute the calls concurrently:

```
[bills, campaigns] = await Promise.all([
      sunriseBills(),
      sunriseCampaigns()
    ]);
```

**Review by agreffard — DISMISSED** [review]

File: backend/src/airtable-api.js (hunk 1 above)
Comment (elliott-king) [reply backend/src/airtable-api.js:148]: Makes sense. Made the change, and re-requesting review.

**Review by agreffard — APPROVED** [review]

**Pull request merged**